cordova-plugin-outline/android/resources/strings/values-el/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8'?>
+<resources>
```

Comment:
Curious: how did you generate these files?

Reply:
I used an ad hoc script. Is it worth cleaning it up to check it in?

Reply:
Definitely :-)